build-files/ncbi-amrfinderplus/4.0.19-2024-12-18.1/Dockerfile

```diff
@@ -0,0 +1,100 @@
+FROM ubuntu:noble AS app
+
+ARG AMRFINDER_VER="4.0.19"
+ARG AMRFINDER_DB_VER="2024-12-18.1"
+ARG BLAST_VER="2.16.0"
+
+LABEL base.image="ubuntu:noble"
+LABEL dockerfile.version="1"
+LABEL software="NCBI AMRFinderPlus"
+LABEL software.version="${AMRFINDER_VER}"
+LABEL description="NCBI resistance gene detection tool"
+LABEL website="https://github.com/ncbi/amr"
+LABEL license="https://github.com/ncbi/amr/blob/master/LICENSE"
+LABEL maintainer="Kelsey Florek"
+LABEL maintainer.email="kelsey.florek@slh.wisc.edu"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL maintainer3="Anders Goncalves da Silva"
+LABEL maintainer3.email="andersgs@gmail.com"
+LABEL maintainer4="Erin Young"
+LABEL maintainer4.email="eriny@utah.gov"
+LABEL maintainer5="Holly McQueary"
+LABEL maintainer5.email="holly.c.mcqueary@mass.gov"
+
+# ncbi-blast+ installed via apt is v2.12.0 - DISABLING so that we can manually install a more recent version
+# see here for reason why I'm manualy installing 2.14.0 instead of using apt-get: https://github.com/ncbi/amr/releases/tag/amrfinder_v3.11.8
+
+# hmmer installed via apt is v3.3.2
```

Comment:
another minor nitpicky suggestion, can you update this to 3.4 or just remove the comment entirely?